src/main/java/no/nav/pto/veilarbportefolje/client/RestClientUtils.java

```diff
@@ -1,17 +1,18 @@
 package no.nav.pto.veilarbportefolje.client;
 
-import no.nav.common.auth.context.AuthContextHolder;
+import no.nav.common.auth.context.AuthContextHolderThreadLocal;
```

Comment:
AuthContextHolderThreadLocal I need to check why it is called ThreadLocal 🤔  does this mean that it is not meant for prod?

Reply:
I dont think its related with env...just its named like that

Reply:
you are correct 👍  but I added a comment on the same issue after a talk with alexander